src/components/Navbar.vue

```diff
@@ -204,7 +204,16 @@
 												text-gray-200
 											"
 											:class="
-												index === searchScrollPosition ? 'even:bg-discord-blurple-630 bg-discord-blurple-660' : ''
+												index === searchScrollPosition
+													? `
+													ring-1
+													ring-gray-200
+													even:bg-discord-blurple-630
+													dark:even:bg-discord-blurple-660
+													bg-discord-blurple-630
+													dark:bg-discord-blurple-660
+													`
+													: ''
```

Comment:
Why isn't this just:
```
index === searchScrollPosition ? 'ring-1 ring-gray-200 even:bg-discord-blurple-630 dark:even:bg-discord-blurple-660 bg-discord-blurple-630 dark:bg-discord-blurple-660' : ''
```

and then let prettier do its formatting?

Reply:
I tried, but prettier doesn't even try to attempt to break up this string. It's happy with it being 200 chars long

Reply:
It's def better than this 🤣